src/Maker/MakeFixtures.php

```diff
@@ -55,8 +55,8 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
         );
 
         $useStatements = new UseStatementGenerator([
```

Comment:
Maybe we can tell static code analysis, that it expects a list?

Reply:
in fact UserStatementGenerator can be used in different ways and the phpdoc seems good :
`/**
     * For use statements that contain aliases, the $classesToBeImported array
     * may contain an array(s) like [\Some\Class::class => 'ZYX']. The generated
     * use statement would appear as "use Some\Class::class as 'ZXY'". It is ok
     * to mix non-aliases classes with aliases.
     *
     * @param string[]|array<string, string> $classesToBeImported
     */
`